pandas/io/formats/style.py

```diff
@@ -1127,40 +1064,35 @@ def bar(self, subset=None, axis=0, color='#d65f5f', width=100,
             - 'mid' : the center of the cell is at (max-min)/2, or
               if values are all negative (positive) the zero is aligned
               at the right (left) of the cell
+        vmin: float or None: minimum bar value, defining the left hand limit
```

Comment:
"or None" -> "optional"

Description should start on the next line.

https://pandas.pydata.org/pandas-docs/stable/contributing_docstring.html